src/components/ContainerSettingsAdvanced.react.js

```diff
@@ -1,8 +1,12 @@
+import _ from 'underscore';
 import React from 'react/addons';
 import metrics from '../utils/MetricsUtil';
 import ContainerUtil from '../utils/ContainerUtil';
 import containerActions from '../actions/ContainerActions';
 
+var ipRegex = /\b(25[0-5]|2[0-4][0-9]|[01]?[0-9][0-9]?)\.(25[0-5]|2[0-4][0-9]|[01]?[0-9][0-9]?)\.(25[0-5]|2[0-4][0-9]|[01]?[0-9][0-9]?)\.(25[0-5]|2[0-4][0-9]|[01]?[0-9][0-9]?)\b/;
+var domainRegex = /([a-z0-9-]+\.(?:com|net|org|co\.uk|localhost|dev))(?:\/|$)/;
```

Comment:
Good suggestion, although a lot looser, but should work nicely - I'll make the proper updates and will push it.